package.json

```diff
@@ -29,9 +29,13 @@
   },
   "devDependencies": {
     "broccoli": "^0.16.9",
+    "broccoli-fixture": "^1.0.0",
```

Comment:
I think I'd generally prefer to use https://github.com/broccolijs/broccoli-test-helper which does provide similar functionality / features, but is a bit more actively maintained. I don't _think_ its a massive change, but feel free to push back if that is too much work here (we can always land this part, then do another PR to migrate to broccoli-test-helper).

Reply:
TIL about broccoli-test-helper! Terrific. Working on this now.

Reply:
PR is updated!